extension/test/unit/config/config.spec.js

```diff
@@ -786,4 +791,19 @@ describe('::config::', () => {
       }
     }
   )
+
+  function deleteExtensionrcFile() {
+    const currentFilePath = fileURLToPath(import.meta.url)
+    const currentDirPath = path.dirname(currentFilePath)
+    const projectRoot = path.resolve(currentDirPath, '../../../')
+    const pathToFile = path.resolve(projectRoot, '.extensionrc')
+
+    // Remove .extensionrc file if already exists
+    fs.stat(pathToFile, function (err, stats) {
+      //ignore error
+      if (!err) {
+        fs.unlinkSync(pathToFile)
```

Comment:
Did you check if we still need to use `.extesionrc `after the the unit test running?

Reply:
Same like config environment variable, We have to create .extensionrc file, If we want to start the application locally.

Reply:
Does it mean if we unlink the file here, the application cannot retrieve the config from it?

Reply:
unlink is to remove the file. This file will be deleted if it exists already.
You have to recreate it again to run the application.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1062/commits/78f8e0dff785f92f911b49eab00d5489d2612e74

I am temporarily renaming the file and naming it back later after the test.
Let me know if you have any better idea here.